velox/exec/RowContainer.h

```diff
@@ -1415,6 +1437,10 @@ class RowContainer {
   // not applicable.
   int32_t probedFlagOffset_ = 0;
 
+  // Maximum size for each variable-length column. If a column is fixed-size,
```

Comment:
What's the value for complex data type which should also take as variable length column

Reply:
Yes, it should be the serialized size.